### DISCUSSION TITLE
Skins lost?

### DISCUSSION THREAD

**Discussion by noiseuu** [SFX-WoW/Masque#49] [Questions] [answered]
After update for 7.3.19 it lost some skins now its called "classic". 
What happend skin calling "blizzard".
Looking like that
![blizzard](https://user-images.githubusercontent.com/43645444/54482092-bbf7ca80-4846-11e9-9993-ea11611cf0a0.png)

**Comment by StormFX** [comment, marked as answer]
It's the same skin. I just renamed it to "Classic" because it wasn't the same as the default UI and I wanted to avoid any legal issues ("Blizzard" is trademarked).
